lib/wyre/wyre_client.dart

```diff
@@ -0,0 +1,37 @@
+import 'package:dio/dio.dart';
+import 'package:mixin_bot_sdk_dart/mixin_bot_sdk_dart.dart';
+
+import '../service/env.dart';
+import 'wyre_api.dart';
+import 'wyre_constants.dart';
+
+class WyreClient {
+  WyreClient._() {
+    _dio = Dio();
+    _dio.options.baseUrl = 'https://$wyreDomain';
+    _dio.options.responseType = ResponseType.json;
+    _dio.interceptors.add(InterceptorsWrapper(
+      onRequest: (
+        RequestOptions options,
+        RequestInterceptorHandler handler,
+      ) async {
+        options.headers['Authorization'] = 'Bearer ${Env.wyreSecret}';
+        handler.next(options);
+      },
+    ));
+    _dio.interceptors.add(MixinLogInterceptor(HttpLogLevel.none));
```

Comment:
Would you assign the headers to _dio.options.headers?

Reply:
it works, but changes to directly assign instead of using the interceptor